generators/Generators.fs

```diff
@@ -1289,6 +1289,42 @@ type Series() =
         |> Option.ofNonErrorObject
         |> Option.renderParenthesized
 
+type SgfParsing() = 
+    inherit GeneratorExercise()
+
+    override __.RenderArrange (canonicalDataCase: CanonicalDataCase) =
+        let rec renderTree (tree: JToken) = 
+            let props = 
+                (tree.SelectToken("properties") :?> JObject).ToObject<Map<string, string[]>>()
+                |> Map.map (fun key value -> List.render value) |> Map.toList
+                |> List.map (fun (key, value) -> sprintf ".Add(\"%s\", %s)" key value)
+                |> String.concat ""
+
+            let children = 
+                if tree.SelectToken("children") <> null then
+                    [| for item in (tree.SelectToken("children"):?> JArray).Children() -> renderTree item |]
+                    |> String.concat "; "
+                else ""
+
+            sprintf "Node (Map.empty%s, [%s])" props children
+
+        seq {
+            if canonicalDataCase.Expected.SelectToken("properties") <> null then
+                yield sprintf "let expected = Some (%s)" (canonicalDataCase.Expected |> renderTree)
+        }
+        |> Seq.toList
+
+    override __.RenderSut (canonicalDataCase: CanonicalDataCase) =
```

Comment:
Overriding this method should not be necessary. Our implementation should follow the canonical data, should we should change the Example implementation to match the canonical data. Could you remove this override?